Library/Formula/camlp5.rb

```diff
@@ -13,20 +13,38 @@ class Camlp5 < Formula
 
   deprecated_option "strict" => "with-strict"
   option "with-strict", "Compile in strict mode"
+  option "with-doc", "Install the html and info documentation"
+  option "with-tex", "Install the pdf, ps, and tex documentation"
 
   depends_on "ocaml"
+  depends_on "gnu-sed" => :build if build.with?("doc") || build.with?("tex")
+  depends_on :tex => [:build, :optional]
+  depends_on "ghostscript" => :build if build.with?("tex") && !File.exist?("/usr/local/bin/ps2pdf")
 
   def install
-    if build.with? "strict"
-      strictness = "-strict"
-    else
-      strictness = "-transitional"
-    end
+    args = ["--prefix", prefix, "--mandir", man]
+    args << "--transitional" if build.without? "strict"
+
+    ps2pdf = which("/usr/local/bin/ps2pdf")
+    ps2pdf ||= which(Formula["ghostscript"].opt_bin/"ps2pdf")
```

Comment:
@DomT4 Better? That would assume ghostscript is made keg_only. If not, then I could use `Formula["ghostscript"].bin/ps2pdf` or may I should just do that either way.